build-system/global-configs/OWNERS

```diff
@@ -9,6 +9,10 @@
           name: 'ampproject/wg-approvers',
           requestReviews: false,
         },
+        {
+          name: 'danielrozenberg',
+          notify: true,
+        },
```

Comment:
Optional: There are [numerous changes](https://github.com/ampproject/amphtml/commits/master/build-system/global-configs) to the prod / canary config files in this directory. If you're primarily interested in `experiments-config.json`, notify yourself for changes to just that file.

Reply:
Done